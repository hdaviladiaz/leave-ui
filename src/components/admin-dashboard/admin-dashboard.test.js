import React from 'react';
import AdminDashboard from './admin-dashboard';
import renderer from 'react-test-renderer';

it('should render AdminDashboard as snapshot', () => {
  const tree = renderer.create(<AdminDashboard/>).toJSON();
  expect(tree).toMatchSnapshot();
});
