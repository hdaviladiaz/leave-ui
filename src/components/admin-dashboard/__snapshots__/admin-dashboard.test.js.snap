// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AdminDashboard as snapshot 1`] = `
<div>
  <div
    className="panel panel-default"
    id={undefined}
  >
    <div
      className="panel-body"
    >
      <div
        className="admin-overview"
      >
        <h4>
          22 junio 2017
        </h4>
        <span
          className="text-muted"
        >
          15
           dias disponibles
        </span>
        <div
          className="admin-overview-box-container"
        >
          <div
            className="admin-overview-box col-md-4"
          >
            <div
              className="admin-overview-box-header"
            >
              Pendientes
            </div>
            <div
              className="admin-overview-box-body"
            >
              <div
                className="admin-overview-box-title"
              >
                4
              </div>
            </div>
          </div>
          <div
            className="admin-overview-box last admin-overview-box-center col-md-8"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="manage-leave-request panel panel-default"
    id={undefined}
  >
    <div
      className="panel-body"
    >
      <h4>
        Solicitudes
      </h4>
      <div
        className="manage-request-table"
      >
        <div
          className="manage-request-table-title"
        >
          Pendientes
        </div>
        <div
          className="manage-request-table-body"
        >
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Hector Davila
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="animate text-success  glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="animate text-danger  glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Juan Perez
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="animate text-success  glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="animate text-danger  glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Alfonso Davila
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="animate text-success  glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="animate text-danger  glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Victor Cruz
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="animate text-success  glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="animate text-danger  glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-no-elements hidden"
          >
            NO HAY ELEMENTOS
          </div>
        </div>
      </div>
      <div
        className="manage-request-table"
      >
        <div
          className="manage-request-table-title"
        >
          Procesadas
        </div>
        <div
          className="manage-request-table-body"
        >
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Hector Davila
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="status text-none glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="status text-danger glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Juan Perez
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="status text-success glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="status text-none glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Alfonso Davila
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="status text-none glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="status text-danger glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-item"
          >
            <div
              className="manage-request-table-item-body"
              onClick={[Function]}
            >
              <div
                className="manage-request-table-item-time"
              >
                6:45 PM
              </div>
              <div
                className="manage-request-table-item-name"
              >
                Victor Cruz
              </div>
            </div>
            <div
              className="manage-request-table-item-tools"
            >
              <span
                className="manage-request-table-item-tools-icon success-icon"
                onClick={[Function]}
                title="Aceptar"
              >
                <span
                  className="status text-success glyphicon glyphicon-ok-sign"
                />
              </span>
              <span
                className="manage-request-table-item-tools-icon"
                onClick={[Function]}
                title="Rechazar"
              >
                <span
                  className="status text-none glyphicon glyphicon-remove-sign"
                />
              </span>
            </div>
          </div>
          <div
            className="manage-request-table-no-elements hidden"
          >
            NO HAY ELEMENTOS
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
